src/sql/workbench/browser/modal/modal.ts

```diff
@@ -485,13 +456,10 @@ export abstract class Modal extends Disposable implements IThemable {
 				} else if (event.equals(KeyCode.Escape)) {
 					DOM.EventHelper.stop(e, true);
 					this.onClose(event);
-				} else if (event.equals(KeyMod.Shift | KeyCode.Tab)) {
-					this.handleBackwardTab(e);
-				} else if (event.equals(KeyCode.Tab)) {
-					this.handleForwardTab(e);
 				}
 			}
 		}));
+		this.disposableStore.add(trapKeyboardNavigation(this._modalDialog!));
```

Comment:
Wouldn't this be an issue since we are adding 2 disposableListeners for `KEY_DOWN` ? Once in line `450` and once from this `trapKeyboardNavigation` call

Reply:
it is ok to have multiple listeners, if you inspect any element using developer tool, you will see most of them have multiple listeners.